Crashlytics/Crashlytics/Controllers/FIRCLSReportManager.m

```diff
@@ -332,7 +333,7 @@ - (FBLPromise *)deleteUnsentReports {
   }
 
   // Grab existing reports
-  BOOL launchFailure = [self checkForAndCreateLaunchMarker];
+  [self checkForAndCreateLaunchMarker];
```

Comment:
What was the reason we made `launchFailure` a class property? With the current code, I think I'd rather make it a local variable just so we don't attempt to check it before `checkForAndCreateLaunchMarker`.

Reply:
It was because the checkForAndCreateLaunchMarker method creates a launch marker, so when I try to use that method again later in the code it always returns true even if there wasn't a class.